Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on make csv-generator
 FROM registry.access.redhat.com/ubi9/ubi-minimal
 LABEL org.kubevirt.hco.csv-generator.v1="/csv-generator"
 
-RUN microdnf update -y && microdnf clean all
+RUN microdnf update -y && microdnf install git -y && microdnf clean all
```

Comment:
What is git needed for in the container image?

Reply:
Referenced in the commit but it's actually a poorly documented dependency of `kustomize` itself:

https://github.com/kubernetes-sigs/kustomize/blob/183928e6baab759f61c033892e32ab5759f442e6/api/internal/git/gitrunner.go#L22-L38